src/openzeppelin/tests/mocks/erc721_panic.cairo

```diff
@@ -0,0 +1,136 @@
+#[contract]
+mod SnakeERC721Panic {
+    use openzeppelin::utils::serde::SpanSerde;
+    use starknet::ContractAddress;
+    use zeroable::Zeroable;
+
+    //
+    // agnostic
+    //
+
+    #[view]
+    fn name() -> felt252 {
+        panic_with_felt252('Some error');
+        3
+    }
+
+    #[view]
+    fn symbol() -> felt252 {
+        panic_with_felt252('Some error');
+        3
+    }
+
+    #[external]
+    fn approve(to: ContractAddress, token_id: u256) {
+        panic_with_felt252('Some error');
+    }
+
+    //
+    // snake
+    //
```

Comment:
We should consider standardizing our comment format. I think there's a few different styles in the PR

Reply:
To be fair, this is also the case with the whole repo

Reply:
we can normalize in a dedicated PR with its associated discussion